encoding/mvt/marshal.go

```diff
@@ -70,6 +70,16 @@ func Marshal(layers Layers) ([]byte, error) {
 		vt.Layers = append(vt.Layers, layer)
 	}
 
+	return vt
+}
+
+// Marshal will take a set of layers and encode them into a Mapbox Vector Tile format.
+// Features that have a nil geometry, for some reason, will be skipped and not included.
+func Marshal(layers Layers) ([]byte, error) {
+	vt, err := MarshalToVectorTile(layers)
```

Comment:
pretty straight forward, i just moved the guts out into another function right before marshalling to bytes. and leave the proto marshaling here for callers of this function